Update introduction.md (#1475)
dbt-labs/docs.getdbt.com: website/docs/docs/introduction.md

dbt also enables analysts to work more like software engineers, in line with the dbt [Viewpoint](viewpoint).

### What is the difference between dbt Core, the dbt CLI and dbt Cloud?

> dbt Core is the software that takes a [dbt project](projects) (`.sql` and `.yml` files) and a command and then creates tables/views in your warehouse. dbt Core includes a command line interface (CLI) so that users can execute dbt commands using a terminal program. dbt Core is [open source](https://github.com/dbt-labs/dbt) and free to use.

Reviewer: ```suggestion
dbt Core is the software that takes a [dbt project](projects) (`.sql` and `.yml` files) and a command and then creates tables/views in your warehouse. You can execute dbt commands from the command line. dbt Core is [open source](https://github.com/dbt-labs/dbt) and free to use.
```

Reviewer: suggs:

- technical accuracy. i don't think we technically ship an actual CLI, do we? 
- consistency. switched from second person (your) to third person (users) in the same paragraph.

Author: thank you so much @nghi-ly ! i don't know about the first bullet - i think it's part of core in some ways - but can I double check with you @jtcohen6 if the below sentence is correct or not:

_dbt Core includes a command line interface (CLI) so that users can execute dbt commands using a terminal program._

Reviewer: @mirnawong1 : thanks for double-checking! i'm curious to know!

Reviewer 2: Thanks for tagging me in @mirnawong1, and sorry for the delay!

> i don't think we technically ship an actual CLI, do we?

@nghi-ly Do you mean this in the sense that Bash, PowerShell, etc would qualify as "actual CLI" (shells)?

If so, fair point! dbt Core can be executed as an application on the Command Line. From reading how other tools describe themselves, I think it's idiomatically correct to say that "dbt Core includes a CLI," in the sense of being able to be used as a CLI app, without implying that dbt is a command line shell.

I'm happy with whatever language you decide on here

Author: thank you @jtcohen6 ! @nghi-ly i'll wait until after coalesce to push this through (unless i can do it now before the docs freeze).

> dbt Cloud is an application that helps teams use dbt. dbt Cloud provides a web-based IDE to develop dbt projects, a purpose-built scheduler, and a way to share dbt documentation with your team. dbt Cloud offers a number of features for free, as well as additional features in paid tiers (check out the pricing [here](https://www.getdbt.com/pricing/)).

Reviewer: ```suggestion
dbt Cloud is a SaaS application that helps teams use dbt. dbt Cloud provides a web-based IDE to develop dbt projects, a purpose-built scheduler, and a way to share your dbt project's documentation with your team. dbt Cloud offers a number of features for free, as well as additional features in paid tiers (check out the pricing [here](https://www.getdbt.com/pricing/)).
```
